addons/binding/org.openhab.binding.dwdunwetter/README_de.md

```diff
@@ -0,0 +1,103 @@
+# DwdUnwetter Binding
+
+Binding zur Abfrage von aktuellen Unwetterwarnungen des Deutschen Wetterdienstes via [DWD Geoserver](https://maps.dwd.de/geoserver/web/ "DWD Geoserver")
+
+## Unterstütztes Thing
+
+Das Binding unterstützt genau ein Thing - Unwetterwarnungen. Ein Thing stellt dabei eine oder mehrere Warnungen für eine Gemeinde bereit.
```

Comment:
Can you also start each sentence on a new line in the lines above this comment?

Reply:
I hope this were the last where I missed it. :)